extensions/projection/json-schema/schema.json

```diff
@@ -85,6 +85,24 @@
                   "maximum": 180
                 }
               }
+            },
+            "proj:shape":{
+              "title":"Shape",
+              "type":"array",
+              "minItems":2,
+              "maxItems":2,
+              "items":{
+                "type":"number"
```

Comment:
Above it's said that the shape consists of integers, so I'd propose to use type: integer.

Reply:
Good idea